src/Components/Components/src/RenderTree/Renderer.cs

```diff
@@ -653,6 +674,21 @@ private void RenderInExistingBatch(RenderQueueEntry renderQueueEntry)
             }
         }
 
+        private static void ProcessAggregateException(List<Exception> exceptions, AggregateException aggregateException)
```

Comment:
Can this be renamed to something clearer? The word "Process" doesn't communicate anything.

Maybe "PopulateListWithInnerExceptions" ?

Reply:
This is gone now